README.md

```diff
@@ -145,6 +145,38 @@ jobs:
           body: ${{ steps.calibre.outputs.markdown }}
 ```
 
+## Compressing images on a schedule
+
+Similar to above, it is also possible to run an addition instance of this action in `compressOnly` mode at specific times, and then raise a new pull request for any images compressed.
+
+```yml
+name: Compress images at 11pm and open a pull request
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '* 23 * * *'
+jobs:
+  build:
+    name: calibreapp/image-actions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+      - name: Compress Images
+        id: calibre
+        uses: calibreapp/image-actions@master
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          compressOnly: true
+      - name: Create New Pull Request

```

Comment:
Is it possible to only create a pull request when there’s images changed? Or is that how the create pull request action works?

Reply:
Yes that's how it works. It doesn't create a PR unless something's changed.

Note this in the README previously in the [Handling pull requests from forked repos](https://github.com/calibreapp/image-actions#handling-pull-requests-from-forked-repos) section (emphasis mine):

> It is also possible to run an addition instance of this action in `compressOnly` mode on pushes to master, and then raise a new pull request for any images commited from a forked repositary pull request. This is shown in the below example which uses the [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub Action to open this new Pull Request (**note this only raises a Pull Request if any files are actually changed in previous steps**).

I can add a similar comment to this "Compressing images on a schedule" section in this PR?

FYI, in case you're curious, this is what it looks like in the Action log (pulled from [this example from my repo]):(https://github.com/HTTPArchive/almanac.httparchive.org/runs/1111523804?check_suite_focus=true)

```
  Pull request branch 'create-pull-request/patch-1600078221' does not exist yet.
  /usr/bin/git checkout --progress -B create-pull-request/patch-1600078221 HEAD
  Switched to a new branch 'create-pull-request/patch-1600078221'
  /usr/bin/git rev-list --right-only --count main...create-pull-request/patch-1600078221
  0
  Branch 'create-pull-request/patch-1600078221' is not ahead of base 'main' and will not be created
  /usr/bin/git branch --delete --force 3339d6f7-5794-4c71-bfc5-f643fb891142
  Deleted branch 3339d6f7-5794-4c71-bfc5-f643fb891142 (was ae815da0).
```

Note this is all on local git in the environment created by the Action so is never pushed to GitHub if there are no changes, so don't worry that the branch is created and then deleted - there's no actual impact of this. Though personally to me it would make sense to do the check first rather than create the branch locally just to delete it! 😁

Reply:
Made this more obvious in https://github.com/calibreapp/image-actions/pull/72/commits/1c1e75d9e20e9d330ff3a1f1a35dbe4bbad54d26

Reply:
Thanks for clarifying ⚡